.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: objective-c
 matrix:
   include:
-    - osx_image: xcode8.3
```

Comment:
Can't we test on xcode8.3?

Reply:
No, since Xcode 8.3 doesn’t support Swift 4.